.github/workflows/build.yaml

```diff
@@ -0,0 +1,26 @@
+name: Docker Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:    
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout source  
+      uses: actions/checkout@v3
+    
+    - name: Login to Docker Hub
+      uses: docker/login-action@v2
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Setup multi-arch build tools
+      uses: docker/setup-buildx-action@v2
+      with:
+        platforms: linux/amd64,linux/arm64,linux/arm/v7
+
+    - name: Multi-arch Docker Build
+      run: |
+        make docker-push
```

Comment:
Any reason why you don't use https://github.com/docker/build-push-action?  IMHO, it's easier to follow a workflow to see what's going on.

And using the https://github.com/docker/metadata-action action by Docker, you also have the option to insert OCI labels with https://github.com/docker/metadata-action.

My attempt: https://github.com/MindTooth/cert-manager-webhook-domeneshop/blob/master/.github/workflows/build-container.yaml

Reply:
I was trying to follow the pattern that was already used in the repository